example/lfp/neuron_lfp_example.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/env/bin python
```

Comment:
Really add a NEURON example here?

Reply:
This is the template from the LFPy people we were tasked with emulating, so I thought it would be useful to include for comparison.

Reply:
Without this though, I wouldn't go to these efforts to match the output plots.

Reply:
Ok, that makes sense